python/lsst/pipe/tasks/calibrate.py

```diff
@@ -73,6 +73,16 @@ class CalibrateConfig(pexConfig.Config):
         doc = "Perform PSF fitting?",
         default = True,
     )
+    doMeasureApCorr = pexConfig.Field(
+        dtype = bool,
+        doc = "Compute aperture corrections?",
+        default = True,
+    )
+    doApplyApCorr = pexConfig.Field(
+        dtype = bool,
+        doc = "Apply aperture corrections? Ignored if doMeasureApCorr is False",
+        default = True,
+    )
```

Comment:
You should add a `validate` method to disallow `doMeasureApCorr=False and doApplyApCorr=True`, since we can't actually do that combination.